graylog2-bootstrap/pom.xml

```diff
@@ -80,21 +80,6 @@
     </dependencies>
 
     <build>
-        <resources>
-            <resource>
-                <!-- git-commit-id-plugin depends on resource filtering to write the
-                    git.properties and version.properties files-->
-                <directory>src/main/resources</directory>
-                <filtering>true</filtering>
-                <includes>
-                    <include>**/*.properties</include>
-                </includes>
-            </resource>
-            <resource>
-                <directory>src/main/resources</directory>
-            </resource>
-        </resources>
-
```

Comment:
It's not only the git-commit-id-plugin that requires resource filtering, so the comment was wrong or at least incomplete in the first place.